### PR TITLE
this is the way task

### PR DESCRIPTION
**Pull request by OdelyaTick22**

### CODE REVIEW

File: ThisIsTheWay.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+
+def way(path):
+    files = []
+    arr = listdir(path)
+    for name in arr:
+        if name.startswith("deep"):
+            files.append(name)
+    return files
+
```
Comment (talbarfm) [inline ThisIsTheWay.py:15]: Add an example of the code execution

File: ThisIsTheWay.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from os import listdir
+
+
+def way(path):
+    files = []
+    arr = listdir(path)
+    for name in arr:
```
Comment (talbarfm) [inline ThisIsTheWay.py]: you can reduce this whole function to be a one linse

File: ThisIsTheWay.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from os import listdir
+
+
+def way(path):
```
Comment (talbarfm) [inline ThisIsTheWay.py]: use type hints and return types

Comment (talbarfm) [inline ThisIsTheWay.py]: document

**Review by talbarfm — CHANGES_REQUESTED** [review]

File: ThisIsTheWay.py [hunk 4]
```diff
@@ -1,11 +1,20 @@
...
-        if name.startswith("deep"):
-            files.append(name)
+    [files.append(name) for name in list(arr) if name.startswith("deep")]
     return files
 
+"""
+An example usage:
+print(way(r"C:\Users\IMOE001\Downloads"))
+>>['deep.txt', 'deepmkvgk.txt']
+"""
```
Comment (lovodkin93) [inline ThisIsTheWay.py]: write the example in a "main"

**Review by lovodkin93 — CHANGES_REQUESTED** [review]

**Pull request merged**